internal/admissionpluginconfig/admissionpluginconfg.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2024 the Pinniped contributors. All Rights Reserved.
```

Comment:
TODO: this new file needs unit tests, and maybe a simple integration test to make sure it works against real Kubernetes clusters.

Reply:
I added unit tests and force pushed.